straxen/url_config.py

```diff
@@ -229,3 +229,8 @@ def format_arg(arg: str, **kwargs):
     '''apply pythons builtin format function to a string
     '''
     return arg.format(**kwargs)
+
+
+@URLConfig.register('interpolatingmap')
+def load_map(some_map):
```

Comment:
InterpolatingMap accepts kwargs, why not have the protocol accept kwargs and pass them along to InterpolatingMap?

Reply:
I did try it, but it doesn't work since the kwargs eventually get passed to [InterpolateAndExtrapolate](https://github.com/XENONnT/straxen/blob/master/straxen/itp_map.py#L15) which doesn't accept kwargs, so got an argument error. I allowed for passing a different `method` though. I guess if someone wants to use a different map we can reconsider this or make a new protocol?